### PR TITLE
Stop writing root to istio-security configmap

### PR DESCRIPTION
**Pull request by howardjohn**
This code is no longer needed, it was a legacy of nodeagent. Now we write to all namespaces in a different configmap

### CODE REVIEW

**Comment by howardjohn** [comment]
/retest 

resource issue?

**Comment by lei-tang** [comment]
Regarding to "Now we write to all namespaces in a different configmap", what is the name of the new configmap?

**Comment by howardjohn** [comment]
something like istio-ca-root-cert or similar? You wrote the code

File: pilot/pkg/bootstrap/istio_ca.go [hunk 1]
```diff
@@ -369,7 +369,7 @@ func (s *Server) createIstioCA(client corev1.CoreV1Interface, opts *CAOptions) (
 		s.caBundlePath = certChainFile
 
 		caOpts, err = ca.NewPluggedCertIstioCAOptions(certChainFile, signingCertFile, signingKeyFile,
```
Comment (costinm) [inline pilot/pkg/bootstrap/istio_ca.go:371]: Every time I see the long list of parameters I want to do something... Worse - there are 4 places where this gets copied. 

For agent I started to create a struct and pass ProxyConfig - most of those things should actually be part of ProxyConfig/MeshConfig anyways.

**Pull request merged**